### PR TITLE
docs/fix(typos): Fix minor typos in English

### PR DESCRIPTION
**Pull request by mribeirodantas**
- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

- Fix minor typos in JS/TS/TSX files
- Fix minor typos in English Markdown files

#Hacktoberfest2023

### CODE REVIEW

File: shared/utils/validate.test.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe('isMicrosoftTranscriptLink', () => {
       expect(isMicrosoftTranscriptLink('https://lean.microsoft.com')).toBe(
         false
       );
-      expect(isMicrosoftTranscriptLink('https://learn.microsft.com')).toBe(
+      expect(isMicrosoftTranscriptLink('https://learn.microsoft.com')).toBe(
```
Comment (jeremylt) [inline shared/utils/validate.test.ts:69]: Double check here - changing urls is always suspicious

**Review by huyenltnguyen — COMMENTED** [review]
LGTM, but your branch appears to be outdated (there is an unresolvable import statement the `settings.test.ts` file).

Please help update the branch with the fCC's `main` branch.

**Comment by Sboonny** [comment]
@mribeirodantas, I have fixed the conflict and merged the latest branch.

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @mribeirodantas, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by huyenltnguyen — APPROVED** [review]

**Pull request merged**